scripts/peer/peer.py

```diff
@@ -0,0 +1,273 @@
+"""
+Test run this with:
+
+    cd <dir where peer.py file is>
+    analysis-runner \
+        --access-level test \
+        --description 'Test run peer analysis' \
+        --output-dir 'peer/2022-03-21' \
+        --dataset tob-wgs \
+        python3 peer.py \
+        --path-to-cell-files 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/expression_files/'
+"""
+
+
+import os
+import logging
+import click
+import hailtop.batch as hb
+import pandas as pd
+from cpg_utils.hail import remote_tmpdir, output_path
+from google.cloud import storage
+
+PEER_DOCKER = 'australia-southeast1-docker.pkg.dev/cpg-common/images/peer:1.3.2'
+SCORES_PATH = 'gs://cpg-tob-wgs-test/kat/pca/nfe_feb22/v0/scores.json'
+COVARIATES_PATH = 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/covariates_files/covariates.tsv'
+SAMPLE_ID_KEYS_PATH = 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/metadata/keys_metadata_sheet.csv'
+
+
+def get_covariates(
+    scores_path, covariates_path, expression_file, sample_id_keys_path
+) -> str:
```

Comment:
This doesn't match the return type

Reply:
I'm a little confused on what would go here instead, since we have two dataframes being returned

Reply:
as you mentioned below, would it look like this?
`-> Tuple[str, str]`

Reply:
I'm getting a pylint error:
`undefined name 'Tuple'`. Not sure if this should be lowercase, or something else?

Reply:
You'll need to import it from typing:

```python
from typing import Tuple
```